src/unum/log/log.c

```diff
@@ -3,6 +3,66 @@
 
 #include "unum.h"
 
+// The log control & configuration for the platform
```

Comment:
Please expand the comment a bit on what the platform should do and help the reader understand how to use the defines, i.e.:
- the structure is the default log subsystem configuration for the platform unless platform defines its own...
- the platform can define its own constants to adjust the defaults without redefining the whole structure...
- list constants that control the behavior and direct reader to the header where they are defined and the explanation is given for each of them...

Reply:
Added enough information.